files/build_templates/docker_image_ctl.j2

```diff
@@ -236,6 +257,11 @@ start() {
         --tmpfs /tmp \
 {%- endif %}
         --tmpfs /var/tmp \
+{%- if docker_container_name == "database" %}
+        --env "NAMESPACE_ID"="$DEV" \
+        --env "NAMESPACE_PREFIX"="$NAMESPACE_PREFIX" \
```

Comment:
Can we use $NET_NS instead of the 2 vars?

Reply:
Thank you for your suggestion, this is my thoughts. --> In the file docker-database-init.sh where I parse these variables, if we pass $NET_NS, I would need to extract the ID out of the string. But then the namespace prefix itself ( now it is "asic" .. but in future ) could be made up of alphabets and digits which could make the ID extraction difficult. So is it ok to leave it like this as 2 vars ?
Also in future .. we could give the option to user to configure the namespace prefix, in case he prefers any name for his platform.